frontend/src/features/admin-form/settings/components/TwilioSettingsSection/TwilioDetailsInputs.tsx

```diff
@@ -192,14 +194,16 @@ export const TwilioDetailsInputs = (): JSX.Element => {
       <Skeleton isLoaded={!isLoading} mt="2.5rem" w="fit-content">
         {hasExistingTwilioCreds ? (
           <Button colorScheme="danger" onClick={onOpen}>
-            Remove and re-enter credentials
+            {t('features.common.removeReenter')}{' '}
+            {t('features.common.entities.credentials')}
           </Button>
         ) : (
           <Button
             isLoading={mutateFormTwilioDetails.isLoading}
             onClick={handleUpdateTwilioDetails}
           >
-            Save credentials
+            {t('features.common.save')}{' '}
+            {t('features.common.entities.credentials')}
```

Comment:
Similar comments as above; to avoid concatenated strings when dealing with translations.